core/frontend-devtools/README.md

```diff
@@ -130,7 +130,7 @@ This package provides several keyins to control the display of background maps,
 
 * `fdt attach maplayer <name>` - Attach a background map layer from name within the map layer source list.  Partial names may be used.
 * `fdt attach mapoverlay <name>` - Attach an overlay map layer from name within the map layer source list.  Partial names may be used.
-* `fdt attach model maplayer [name] [globe|reality]` - Attach a model map layer for each unique model of the currently selected elements to either the globe or attached reality models. If `globe` or `reality` is not specified, the attachment will default to `globe`. If `name` is specified, use that. If `name` is specified as `0`, that is considered `undefined` or unspecified.
+* `fdt attach model maplayer [name] [globe|reality|imodel] [modelId0,modelId1,...modelIdN]` - Attach a model map layer for each unique model of the currently selected elements to either the globe, attached reality models, or iModel. If `globe`, `reality`, or `imodel` is not specified, the attachment will default to `globe`. If `name` is specified, use that. If `name` is specified as `0`, that is considered `undefined` or unspecified. `modelIds` is a list of models within an iModel to drape onto, given that `imodel` is specified as the drape target; the list is separated by commas and must have no spaces.
```

Comment:
What are the correct arguments to run this command with the test dataset you provided? I am running `fdt attach model maplayer imodel 80,82,84` and not seeing anything (I got these model IDs by turning on tooltips when viewing the iModel, and converting them to decimal). And are there any other steps to reproduce the draping, like selecting models first?

Reply:
`fdt attach model maplayer 0 imodel` while an element in the model you want to drape is selected.

You need to specify the [name] argument if you are specifying the globe|reality|imodel argument. `0` means `undefined`.

Based on your comment, I realized I neglected to update the documentation here -- because draping now occurs onto _**all**_ models, not a list of drape targets.  I have removed that from this doc file.

Reply:
Thanks, with that command I was able to reproduce.

I know this was in the original docs for this key-in before this PR, but the `name` argument refers to a name in the map layer source list which is to be attached, right? The wording for it is confusing. So is there any case where you would use `fdt attach model maplayer [name] imodel` with a name other than 0, or would that not make sense because you can't specify the "name" of the element you want to drape?

Reply:
Yeah, the `name` argument was not really documented in the original version of this tool (before this PR).  [Here is how it is used in the code](https://github.com/iTwin/itwinjs-core/blob/13f8570ae530ddcf3cb29b29ca0c18e207a714c6/core/frontend-devtools/src/tools/MapLayerTool.ts#L92).  [According to this documentation](https://github.com/iTwin/itwinjs-core/blob/13f8570ae530ddcf3cb29b29ca0c18e207a714c6/core/common/src/MapLayerSettings.ts#L145), `name` is a "user-friendly name for the layer."  Is that a full enough description sense according to your experience with map layers?

We probably should call this out in the keyin, whatever the best description is.

Reply:
Ah ok, I was misunderstanding what that argument is for. Then I guess I would suggest changing the sentence to:
"If `name` is specified, use that as the user-friendly name for the new map layer." just for clarity's sake (this is unrelated to model draping specifically).